### PR TITLE
Improvement/90 improve android keyboard

### PR DESCRIPTION
**Pull request by Cal-L**
**Description**

Remove learning from Android keyboards in both app and webview.

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**
https://github.com/MetaMask/mobile-planning/issues/90

Resolves #???

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -251,7 +255,8 @@ dependencies {
 	implementation 'androidx.multidex:multidex:2.0.1'
 	implementation 'androidx.annotation:annotation:1.1.0'
 	implementation 'androidx.appcompat:appcompat:1.2.0'
-	implementation "com.facebook.react:react-native:+"  // From node_modules
+	implementation project(':ReactAndroid')
+	// implementation "com.facebook.react:react-native:+"  // From node_modules
```
Comment (sethkfman) [inline android/app/build.gradle:259]: remove commented line

**Review by sethkfman — COMMENTED** [review]
This looks good with a minor comment. Do you need someone to walk through text fields and do double check?

**Comment by sethkfman** [comment]
Before this PR is completed we need to update the Bitrise build to make sure the NDK will work.

In the Bitrise.yml we will need something like:
Step Update
`
            - install-missing-android-tools@2:
                  inputs:
                      - ndk_revision: $NDK_VERSION
                      - gradlew_path: $PROJECT_LOCATION/gradlew
`

Add variable 
`
        - opts:
            is_expand: false
          NDK_VERSION: 17b
`

**Comment by Cal-L** [comment]
Closing. Refer to this instead:
https://github.com/MetaMask/metamask-mobile-ghsa-rc56-6qmj-qg85/pull/1

**Pull request closed without merging**